index.html

```diff
@@ -8,41 +9,343 @@
-        </nav>
+                    <img class="menu" src="./icons/menu.png" alt="menu" height="25" width="25">
+
+                </div>
+            </nav>
         </header>
-        
+
         <div class="landing-section">
-            
+
             <section class="landing-text">
-                
+
                 <h1>Hey there. I’m <br>Waleed<br>
                 </h1>
                 <h2 id="landingtext">I’m a software <br>developer</h2>
                 <p id="landingtext2">
-                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
                     don’t hestiate to contact me.<br>
-                    <button type="button" class="connect">Lets Connect!
-                    </button>
-                </p>     
+                    <!-- <button type="button" class="connect">Lets Connect!
+                    </button> -->
+                </p>
                 <ul class="social-media-icons">
-                    <li class="social-media"><a href="https://twitter.com/" ><img src="./icons/Social_icon1.png"  alt="Twitter" ></a></li> 
-                    <li class="social-media"><a href="https://www.linkedin.com/" ><img src="./icons/Social_icon2.png"     alt="Linkedin" ></a></li> 
-                    <li class="social-media"><a  href="https://medium.com/" ><img src="./icons/Social_icon3.png"      alt="Medium" ></a></li> 
-                    <li class="social-media"><a href="https://angel.co/" ><img src="./icons/Social_icon5.png"      alt="angellist"></a></li>
-              </ul>          
+                    <li class="social-media"><a href="https://twitter.com/"><img src="./icons/Social_icon1.png"
+                                alt="Twitter"></a></li>
+                    <li class="social-media"><a href="https://www.linkedin.com/"><img src="./icons/Social_icon2.png"
+                                alt="Linkedin"></a></li>
+                    <li class="social-media"><a href="https://medium.com/"><img src="./icons/Social_icon3.png"
+                                alt="Medium"></a></li>
+                    <li class="social-media"><a href="https://angel.co/"><img src="./icons/Social_icon5.png"
+                                alt="angellist"></a></li>
+                </ul>
             </section>
         </div>
     </div>
+    <div class="container2">
+        <a href="#main-heading"><img class="arrow" src="./icons/down-arrow.svg" alt="Arrow"></a>
+        <h2 id="main-heading">My Recent <br>WORKS</h2>
+        <img id="indicator" src="./icons/Indicator.svg" alt="indicator">
+    </div>
+
+    <div class="card-holder">
+
+        <div class="card-holder-item">
+
+            <div class="holder-image">
```

Comment:
this also should be an `<img>` tag, I suggest to do that

Reply:
Hey, can you please describe a little bit more what you want me to do? if you want me to add <img>tag inside the <div> or you want me to replace the <div> with <img> tag? (if i replace <img> tag with <div> it leaves border which looks untidy so thats why its not possible to use img and also i cannot leave it without putting image and if i put any image in img tag that wont match the template)

Reply:
@youmari

Reply:
@caasperr I think you are okay right now but i suggested you change the div tag to img tag